marimo/_ast/app.py

```diff
@@ -459,43 +295,45 @@ def _run_cell_sync(
         output, defs = self._runner.run_cell_sync(cell._cell.cell_id, kwargs)
         return output, _Namespace(defs, owner=self)
 
-    def _register_ui_element_update(self, value: UIElement[Any, Any]) -> None:
-        if value not in self._pending_ui_element_updates:
-            self._pending_ui_element_updates.append(value)
+    async def _set_ui_element_value(
+        self, request: SetUIElementValueRequest
+    ) -> bool:
+        app_kernel_runner = self._get_kernel_runner()
+        return await app_kernel_runner.set_ui_element_value(request)
 
-    def _mime_(self) -> tuple[KnownMimeType, str]:
+    async def embed(self) -> AppEmbedResult:
```

Comment:
is this public? worth adding docs? im not sure they show up in the docs yet, but maybe we should add it?